test/test-fs.c

```diff
@@ -2662,3 +2662,104 @@ TEST_IMPL(fs_read_write_null_arguments) {
 
   return 0;
 }
+
+static void thread_exec(int fd, char* data, int size, int interval, int doread) {
+  pid_t pid=getpid();
```

Comment:
Nah, discard the Windows thing, just add some ifdefs to skip the test on Windows.